esp/scm/ws_workunits.ecm

```diff
@@ -347,9 +347,10 @@ ESPrequest [nil_remove] WUQueryRequest
     string LogicalFile;
     string LogicalFileSearchType("");
 
-    string ApplicationName;
-    string ApplicationKey;
-    string ApplicationData;
+    [depr_ver("1.54")] string ApplicationName;
+    [depr_ver("1.54")] string ApplicationKey;
+    [depr_ver("1.54")] string ApplicationData;
```

Comment:
I think you want depr_ver("1.55").
I think depr_ver needs the version for which the field becomes deprecated.. in this case depr_ver("1.55").

deper_ver means allow "version < value"
max_ver means allow "version <= value"